x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/util/Check.java

```diff
@@ -36,4 +36,14 @@ public static void notNull(Object object, String message, Object... values) {
             throw new SqlIllegalArgumentException(message, values);
         }
     }
+    
+    public static void isNumberOutOfRange(Object object, String objectName, Long from, Long to) {
+        if (!(object instanceof Number)) {
+            throw new SqlIllegalArgumentException("A number is required; received [{}]", object);
+        }
+        if (((Number) object).longValue() > to || ((Number) object).longValue() < from) {
+            throw new SqlIllegalArgumentException("[{}] must be in the interval [{}..{}], but was [{}]", 
```

Comment:
Optional:
```suggestion
            throw new SqlIllegalArgumentException("[{}] has the value [{}] out of allowed range [{}..{}]", 
```
or similar (and respective params swapped).

Reply:
Thanks for the advice.